compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -285,6 +261,115 @@ class CompilationTests extends ParallelTesting {
       } :: Nil
     }.map(_.checkCompile()).foreach(_.delete())
   }
+
+  @Test def bytecodeIdemporency: Unit = {
+    var failed = 0
+    var total = 0
+    val blacklisted = Set(
+      "pos/Map/scala/collection/immutable/Map",
+      "pos/Map/scala/collection/immutable/AbstractMap",
+      "pos/t1203a/NodeSeq"
```

Comment:
Why is there a blacklist? If these files fail the test, shoudn't issues be filed for them?